todo/ios/MongoDBSample/Classes/Controllers/Authentication/AuthenticationViewController.swift

```diff
@@ -76,12 +82,13 @@ class AuthenticationViewController: UIViewController, GIDSignInUIDelegate, GIDSi
                 authAvailable = true
                 
                 var configureError: NSError?
-                GGLContext.sharedInstance().configureWithError(&configureError)
+                //GGLContext.sharedInstance().configureWithError(&configureError)
```

Comment:
should this be commented out?

Reply:
Looks like my latest changes didn't get pushed. :(